content/english/hpc/architecture/isa.md

```diff
@@ -21,7 +22,7 @@ An ISA essentially defines how the hardware should interpret the machine languag
 Historically, there have been many competing ISAs in use. But unlike [character encodings and instant messaging protocols](https://xkcd.com/927/), developing and maintaining a completely separate ISA is costly, so mainstream CPU designs ended up converging to one of the two families:
 
 - **Arm** chips, which are used in almost all mobile devices, as well as other computer-like devices such as TVs, smart fridges, microwaves, [car autopilots](https://en.wikipedia.org/wiki/Tesla_Autopilot), and so on. They are designed by a British company of the same name, as well as a number of electronics manufacturers including Apple and Samsung.
-- **x86**[^x86] chips, which are used in almost all servers and desktops, with a few notable exceptions such as Apple's M1 MacBooks, AWS's Graviton processors, and the current [world's fastest supercomputer](https://en.wikipedia.org/wiki/Fugaku_(supercomputer)), all of which use Arm-based CPUs. They are designed by a duopoly of Intel and AMD.
+- **x86**[^x86] chips, which are used in almost all servers and desktops, with a few notable exceptions such as Apple's M1 MacBooks, AWS's Graviton processors, and the current [world's fastest supercomputer](https://en.wikipedia.org/wiki/Fugaku_(supercomputer)), all of which use x86-based CPUs. They are designed by a duopoly of Intel and AMD.
```

Comment:
The sentence is a bit confusing, but not wrong: M1, Graviton, and Fugaku (now second fastest) use Arm.

Perhaps it would be clearer to rearrange a bit:

```markdown
 **Arm** chips, which are used in almost all mobile devices, as well as other computer-like devices such as TVs, smart fridges, microwaves, [car autopilots](https://en.wikipedia.org/wiki/Tesla_Autopilot). They are also powering Apple's M1 MacBooks, AWS's Graviton processors, and the [world's second fastest supercomputer](https://en.wikipedia.org/wiki/Fugaku_(supercomputer)). Arm chips are designed by a British company of the same name, as well as a number of electronics manufacturers, including Apple and Samsung.
- **x86**[^x86] chips, which are used in almost all servers and desktops, with a few notable exceptions based on Arm. They are designed by a duopoly of Intel and AMD.
```